src/ZodError.ts

```diff
@@ -129,14 +132,19 @@ export type ZodFormattedError<T> = {
   ? { [K in keyof T]?: ZodFormattedError<T[K]> }
   : unknown);
 
+export interface ZodErrorOpts {
```

Comment:
@MrAwesome 

There are still a few places (mostly in variable assignments) that reference `opts` instead of `params`. For consistency maybe it's worth cleaning those up, too? As far as I can tell from the existing codebase, `opt` is only ever part of `option` or `optional`.